docs/releases/3.1.txt

```diff
@@ -292,7 +292,7 @@ Internationalization
   (string) value to explicitly state that the cookie is sent with all same-site
   and cross-site requests.
 
-* Added support and translations for the Algerian Arabic language.
+* Added support and translations for the Algerian Arabic and Kyrgyz languages.
```

Comment:
Maybe we should use `Kirghiz`.

Reply:
[https://en.wikipedia.org/wiki/Kyrgyz_people](Kyrgyz_people, wiki), It would raise a lot of misunderstanding and chaos.

Reply:
Just asking, because `en-US` spelling checker prefers `Kirghiz`.